graylog2-server/src/main/java/org/graylog2/plugin/configuration/fields/ConfigurationField.java

```diff
@@ -42,4 +42,8 @@
     List<String> getAttributes();
 
     Map<String, Map<String, String>> getAdditionalInformation();
+
+    default int getPosition() {
+        return 100;
```

Comment:
I think we should return `DEFAULT_POSITION` here instead.

Reply:
Thanks! that was a sloppy oversight :/